specifyweb/frontend/js_src/lib/components/Merging/Status.tsx

```diff
@@ -94,6 +95,15 @@ export function Status({
       dimensionsKey="merging-progress"
       header={statusLocalization[state.status]}
       onClose={undefined}
+      icon={
+        state.status === 'PENDING' ? (
+          icons.cog
+        ) : state.status === 'MERGING' ? (
+          <span className="text-red-500">{icons.exclamationCircle}</span>
+        ) : (
+          <span className="text-green-500">{icons.checkCircle}</span>
```

Comment:
Not the biggest fan of hardcoding the colors here. There are considerations with light/dark mode, and accessibility concerns to those that are color blind. 
If the colors are important, can you use a `dialogIcon` then, to stay consistent with the rest of Specify?
https://github.com/specify/specify7/blob/2a259ca22b439ba71b9313705cb2a253ac8edd6e/specifyweb/frontend/js_src/lib/components/Atoms/Icons.tsx#L111-L124

Reply:
@melton-jason great suggestion!

the only thing I can say in defense is that the `-500` colors are the only colors that work fine both in light and dark mode (though fine does not mean ideal)